docs/API.md

```diff
@@ -352,6 +354,17 @@ Definition for a security/complexity requirement for application passwords.
 -   **regex**: _`RegExp`_
     -   A regular expression the defines the complexity requirement (e.g., `/[A-Z]+/`)
 
+## RegistrationConfig
+
+Custom configuration for registration workflows.
+
+### Type Declaration
+
+-   **firstNameLengthLimit** (optional): _`number`_
+    -   Character limit for the first name text field in the registration workflows.
+-   **lastNameLengthLimit** (optional): _`number`_
+    -   Character limit for the last name text field in the registration workflows.
```

Comment:
I'm curious about the format here and future-proofing.

What do you think about something like:
```
registrationConfig: {
    firstName?: {
        maxLength?: number;
    },
    lastName?: {
        maxLength?: number;
    }
}
```

so we could add additional fields or constraints (e.g., regex, isRequired, minLength, inputType) without blowing up a flat list of properties?

Reply:
good call